src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -0,0 +1,135 @@
+import datajoint as dj
+import numpy as np
+from ripple_detection import get_multiunit_population_firing_rate
+
+from spyglass.common import Session  # noqa: F401
+from spyglass.spikesorting.spikesorting_merge import SpikeSortingOutput
+from spyglass.spikesorting.unit_inclusion_merge import UnitInclusionOutput
+from spyglass.utils.dj_mixin import SpyglassMixin
+
+schema = dj.schema("sorted_spikes_group_v1")
+
+
+@schema
+class SortedSpikesGroup(SpyglassMixin, dj.Manual):
+    definition = """
+    -> Session
+    sorted_spikes_group_name: varchar(80)
```

Comment:
In general, I avoid redundancy of `TableA.fetch('table_a_field')`, but the counter-arg is that it then requires projection when it gets inherited elsewhere, `TableB.fetch('{table_x_}field')` in the rare case where it X could be A or B

```suggestion
    group_name: varchar(80)
```